src/adapter/src/rbac.rs

```diff
@@ -204,15 +214,35 @@ fn generate_required_plan_attribute(plan: &Plan) -> Option<Attribute> {
         Plan::DropObjects(plan) if plan.object_type == ObjectType::Role => {
             Some(Attribute::CreateRole)
         }
-        Plan::CreateSource(_)
-        | Plan::CreateSources(_)
-        | Plan::CreateTable(_)
+        Plan::CreateSource(CreateSourcePlan { cluster_config, .. })
```

Comment:
Is this diff related to required privs? It looks like it's related to the test below, and they could be in another commit?

Reply:
Yes, this should probably go in another commit. I was just being lazy. I'll extract this into a separate PR.